### DISCUSSION TITLE
Displaying an ordered list after a code block, with code annotations enabled?

### DISCUSSION THREAD

**Discussion by psig-bah** [squidfunk/mkdocs-material#3294] [Q&A] [answered]
Enjoying cleaning up a lot of footnotes with the new code annotations feature (thank you!), but noticed on the few pages where I've got an unassociated ordered list following a code block, it's getting hidden when the page loads.

For example:

`````markdown
# My Docs Page

```bash
sh ./hello_world.sh
```

After running the script, complete the following steps:

1.  Go to Step 2
2.  Skip Step 3
3.  ???
4.  Profit
`````

When the page loads:
* the ordered list is not displayed
* it **is** present in the page source
* the list items are accessible as code annotations in the code block (i.e. adding a `# (3)` at the end of the `sh` line will render a `???` code annotation)

I noticed putting a new header break between the code block and the ordered list will get it to show up again, but wanted to check if there's a cleaner workaround, or formatting option I missed?

(using the 8.0.3 docker image, with `- content.code.annotate` set in my mkdocs.yml file)

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. It's easy to do the other way round, by explicitly marking each code block you want to include with the `.annotate` CSS class and not enabling it globally. This is also explained in the little box in the [code annotations documentation](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-annotations).

However, I think it's quite sensible to say that if we find no annotation at all in the code, we can leave the code block as-is. You can open a feature request if you like.

**Reply by squidfunk** [reply]
Fixed in 87df85def – no more need to open an issue. If the code block contains no annotation markers, the list will remain untouched.

**Reply by squidfunk** [reply]
The fix was just released as part of 8.0.4.

**Reply by psig-bah** [reply]
awesome, thank you!
